pkg/minikube/cluster/cluster.go

```diff
@@ -214,7 +215,13 @@ func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 	fmt.Println(fmt.Sprintf("Downloading ISO '%s'", m.MinikubeISO))
 
 	// store the iso inside the MINISHIFT_HOME dir
-	response, err := http.Get(m.MinikubeISO)
+	myClient := http.DefaultClient
```

Comment:
Code duplication. This is a candidate for a general download function that can be part of our utility library

```
Get(uri string, proxy ProxyInfo) error {}
```